libs/ActivityKit/source/activities/DisplayTags.cpp

```diff
@@ -0,0 +1,36 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+// LCOV_EXCL_START
+
+#include "DisplayTags.h"
+
+namespace leka::activity {
+
+void DisplayTags::start()
+{
+	_videokit.displayImage("fs/home/img/system/robot-misc-robot-misc-screen_empty_white.jpg");
+
+	_backup_callback = _rfidkit.getCallback();
+
+	auto on_tag_detected_callback = [this](const MagicCard &card) {
+		if (card == MagicCard::remote_standard) {
+			stop();
+		} else {
+			snprintf(_path_buffer.data(), _path_buffer.size(), "fs/home/img/id/00%x.jpg", card.getId());
```

Comment:
ici le format n'est pas évolutif.

si tu as `0x0100` ça marche déjà plus.

il faut un truc du genre:

```suggestion
			snprintf(_path_buffer.data(), _path_buffer.size(), "fs/home/img/id/0x%.4X.jpg", card.getId());
```

Reply:
`x` si minuscule et `X` si majuscule dans les noms de fichiers